components/org.wso2.micro.integrator.initializer/src/main/java/org/wso2/micro/integrator/initializer/StartupFinalizer.java

```diff
@@ -68,7 +75,41 @@ public void cleanup() {
         /*listerManagerServiceRegistration.unregister();*/
     }
 
+    /**
+     * Function to add CApp deployer
```

Comment:
Please refer to the following to learn about how to write proper doc comments.

https://www.oracle.com/technetwork/java/javase/documentation/index-137868.html

Reply:
Done.